aip/0235.md

```diff
@@ -0,0 +1,86 @@
+---
+aip:
+  id: 235
+  state: draft
+  created: 2019-06-18
+permalink: /235
+redirect_from:
+  - /0235
+---
+
+# Batch Delete
+
+Some APIs need to allow users to delete a set of resources in a single transaction. A batch delete method provides this functionality.
+
+## Guidance
+
+Batch delete methods are specified using the following pattern:
```

Comment:
```suggestion
APIs **may** support Batch Delete using the following pattern:
```

Reply:
I will incorporate this in my own commit